ocfweb/docs/docs/contact/slack.md

```diff
@@ -6,18 +6,50 @@ We also offer IRC as a chat service. If that is what you were looking for head o
 
 ### Join the OCF Slack Workspace
 
-Joining the OCF Slack requires an OCF account. If you don't have one already,
-you can create one [here][join]. Head over to the [OCF Slack][slack] and create
-an account with your <ocfusername>@ocf.berkeley.edu email address. This will
-*only* work with an OCF email address. Note that your OCF email by default will
-forward to your CalNet email address (e.g. oski@berkeley.edu). Using the
-confirmation link in your email, you should now be able to set a display name
-and password and join the OCF workspace. We recommend setting your display name
-to your OCF username for consistency.
+1. Create an OCF account [here][join].
+
+2. Go to the OCF Slack workspace [here][slack] and click the "If you have an
+@ocf.berkeley.edu email address, you can create an account." button.
+
+3. Enter your OCF username in the box, this should send an email your Berkeley
+   email. If you don't recieve an email, please contact a staff member for
+   assistance.
+
+4. Click the link in your Berkeley email and follow the instructions to complete
+   setup. We recommend setting your display name to your OCF username for
```

Comment:
I'd go with "strongly recommend" honestly, but very much a nitpick, so feel free to disregard